test/statetest/statetest.cpp

```diff
@@ -80,18 +81,21 @@ int main(int argc, char* argv[])
             ->required()
             ->check(CLI::ExistingPath);
 
-        bool trace_flag = false;
-        app.add_flag("--trace", trace_flag, "Enable EVM tracing");
+        bool trace = false;
+        bool trace_summary = false;
+        const auto trace_opt = app.add_flag("--trace", trace, "Enable EVM tracing");
+        app.add_flag("--trace-summary", trace_summary, "Output trace summary only")
+            ->excludes(trace_opt);
```

Comment:
Does `excludes()` disallow having both options at once? I would allow it, given that only `--trace` is actually the same as `--trace --trace-summary`

Reply:
I'm not sure. See `--trace --trace-summary` you can infer both are needed.

Reply:
I prefer more restrictive version. We can change it when there is a complain.